test/unit/document_spec.js

```diff
@@ -213,12 +213,12 @@ describe("document", function () {
       const acroForm = new Dict();
 
       let pdfDocument = getDocument(acroForm);
-      let fields = await pdfDocument.fieldObjects;
+      let fields = (await pdfDocument.fieldObjects) || null;
```

Comment:
I don't believe that this is actually necessary, besides we should be testing what the method actually returns.
```suggestion
      let fields = await pdfDocument.fieldObjects:
```